stixpy/coordinates/tests/test_frames.py

```diff
@@ -0,0 +1,82 @@
+import astropy.units as u
+import numpy as np
+import pytest
+from astropy.coordinates import SkyCoord
+from astropy.wcs import WCS
+from sunpy.coordinates.frames import HeliographicStonyhurst, Helioprojective
+from sunpy.map import Map, make_fitswcs_header
+
+from stixpy.coordinates.frames import STIXImaging, stix_frame_to_wcs, stix_wcs_to_frame
+from stixpy.map.stix import STIXMap
+
+
+@pytest.fixture
+def stix_wcs():
+    w = WCS(naxis=2)
+
+    w.wcs.dateobs = '2024-01-01 00:00:00.000'
+    w.wcs.crpix = [10, 20]
+    w.wcs.cdelt = np.array([2, 2])
+    w.wcs.crval = [0, 0]
+    w.wcs.ctype = ["SXLN-TAN", "SXLT-TAN"]
+
+    w.wcs.aux.hgln_obs = 10
+    w.wcs.aux.hglt_obs = 20
+    w.wcs.aux.dsun_obs = 1.5e11
+
+    return w
+
+
+@pytest.fixture
+def stix_frame():
+    obstime = '2024-01-01'
+    observer = HeliographicStonyhurst(10 * u.deg,
+                                      20 * u.deg,
+                                      1.5e11 * u.m,
+                                      obstime=obstime)
+
+    frame_args = {'obstime': obstime,
+                  'observer': observer,
+                  'rsun': 695_700_000 * u.m}
+
+    frame = STIXImaging(**frame_args)
+    return frame
+
+def test_stix_wcs_to_frame(stix_wcs):
+    frame = stix_wcs_to_frame(stix_wcs)
+    assert isinstance(frame, STIXImaging)
```

Comment:
Should the values in the returned frame object be tested to ensure the function is parsing the WCS correctly?